src/proxy/ddl.go

```diff
@@ -192,6 +192,8 @@ func (spanner *Spanner) handleDDL(session *driver.Session, query string, node *s
 		if err := route.CreateDatabase(database); err != nil {
 			return nil, err
 		}
+		// If some error happens in mysql internal error, we should check and remove the
```

Comment:
Fix the [issue](https://github.com/radondb/radon/issues/484)，and tell us the correct operation if the err happens.